mmv1/third_party/terraform/services/storage/resource_storage_bucket.go.erb

```diff
@@ -1254,6 +1259,14 @@ func flattenBucketLifecycleRuleCondition(condition *storage.BucketLifecycleRuleC
 			ruleCondition["with_state"] = "ARCHIVED"
 		}
 	}
+  // setting no_age value from state config since it is terraform only variable and not getting value from backend.
+	if v, ok := d.GetOk("lifecycle_rule.0.condition"); ok{
+    state_condition := v.(*schema.Set).List()[0].(map[string]interface{})
+    ruleCondition["no_age"] = state_condition["no_age"].(bool)
+	} else {
+	  ruleCondition["no_age"] = false
```

Comment:
I'm not sure if this is necessary. Do we need to explicitly assign a value of false when there is no condition specified?